python/main.py

```diff
@@ -39,4 +74,19 @@ async def get_image(image_filename):
         logger.debug(f"Image not found: {image}")
         image = images / "default.jpg"
 
-    return FileResponse(image)
\ No newline at end of file
+    return FileResponse(image)
+
+@app.get("/items/{item_id}")
+def get_itemsid(item_id:int):
+    try:
+        with open("items.json", "r") as f:
+            mydata = json.load(f)
+            return mydata["items"][item_id]
+    except KeyError:
+        raise HTTPException(
+            status_code=404, detail="'items' not EXIST"
+        )
```

Comment:
Current codes generate json file with 'items' key, so if there isn't items in json file it means there are some issues in server side.
In this case, just ignoring KeyError (deleting L85 to L88) will be enough, because fastapi handle unexpected exception and return 500 Internal server error to the client.